<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>346</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Song Creator</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Song Title:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Release Year:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Is instrumental?</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Song Ratings:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QSpinBox" name="ratingBox1"/>
        </item>
        <item>
         <widget class="QSpinBox" name="ratingBox2"/>
        </item>
        <item>
         <widget class="QSpinBox" name="ratingBox3"/>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Duration (in minutes):</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="instrumentalCBox">
        <item>
         <property name="text">
          <string>Non-Instrumental</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Instrumental</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="durationBox"/>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="releaseBox">
        <property name="maximum">
         <number>2030</number>
        </property>
        <property name="value">
         <number>2000</number>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="titleEdit">
        <property name="inputMask">
         <string extracomment="Input name"/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="initListButton">
        <property name="text">
         <string>Initializer List Init</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="variablesButton">
        <property name="text">
         <string>Variables Init</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pointerButton">
        <property name="text">
         <string>Pointer Init</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="referenceButton">
        <property name="text">
         <string>Reference Init</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
